src/flag_gems/ops/cross_entropy_loss.py

```diff
@@ -208,8 +210,9 @@ def celoss_indice_smooth_kernel(
         _sum += log * smooth * w[:, None]
 
     out = tl.sum(_sum, axis=0)
+    out = tl.where(ignore_mask, out, 0)
     out_ptrs = out_ptr + pid_n * D + offset_d
-    tl.store(out_ptrs, out, mask=tgt_mask and ignore_mask)
+    tl.store(out_ptrs, out)
```

Comment:
Also, I think masking is needed here, or it would access illegal memory.

Reply:
done